hawkular/hawkular_api.py

```diff
@@ -107,5 +107,99 @@ def get_hawkular_server_groups(self, feed_id):
 
         return server_groups
 
-    def generic_for_testing(self):
-        return self.__hawkular__.list_server()
+    def get_port(self):
+        try:
+            return self.__hawkular__.port
+        except:
+            self.__exception_handler()
+
+    def get_ip_address(self):
+        try:
+            return self.__hawkular__.get_ip_address
+        except:
+            self.__exception_handler()
+
+        def generic_for_testing(self):
+            return self.__hawkular__.list_server()
+
+    ''' Alerts '''
+
+    def get_alert_list_event(self):
+        # TO-DO: Return formatted Dict
+        try:
+            return self.__hawkular__.alert.list_event()
+        except:
+            self.__exception_handler()
+
+    def get_alert_auth(self):
```

Comment:
There is no separate auth, tenentId, hostname, etc for alerts, metrics. Everything passed from main `__hawkular__` object.

You can get everything from main object. https://github.com/ManageIQ/wrapanapi/blob/master/mgmtsystem/hawkular.py#L164